File: abc/abc342/a.cs
// https://atcoder.jp/contests/abc342/submissions/50627356

using static System.Console;

public class A
{
  static void Main()
  {
    string s = ReadLine();
    for(int i = 0; i < s.Length; i++)
    {
      bool check = true;

      for(int j = 0; j < s.Length; j++)
      {
        if(i == j) continue;
        if(s[i] == s[j])
        {
          check = false;
          break;
        }
      }

      if(check)
      {
        Write(i + 1);
        break;
      }
    }
  }
}

File: abc/abc342/a.java
// https://atcoder.jp/contests/abc342/submissions/50627292

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    String s = stdIn.next();
    int[] cnt = new int[26];
    for(int i = 0; i < 26; i++) {
      cnt[i] = 0;
    }

    for(int i = 0; i < s.length(); i++) {
      cnt[s.charAt(i) - 'a']++;
    }

    for(int i = 0; i < s.length(); i++) {
      if(cnt[s.charAt(i) - 'a'] == 1) {
        System.out.print(i + 1);
        break;
      }
    }
  }
}
